install/0000_30_machine-api-operator_04_metal3provisioning.crd.yaml

```diff
@@ -0,0 +1,148 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: provisionings.metal3.io
+spec:
+  group: metal3.io
+  names:
+    kind: Provisioning
+    listKind: ProvisioningList
+    plural: provisionings
+    singular: provisioning
+  scope: Cluster
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      description: 'Provisioning contains configuration used by the Provisioning
+        service (Ironic) to provision baremetal hosts. \n Provisioning is created
+        by the Openshift installer using admin or user provided information about
+        the provisioning network and the NIC on the server that can be used to PXE
+        boot it. \n This CR is a singleton, created by the installer and currently
+        only consumed by the machine-api-operator to bring up and update containers
+        in a metal3 cluster.'
+      type: object
+      required:
+      - spec
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: Specification of provisioning configuration for Metal3.
+          type: object
+          properties:
+            provisioningInterface:
+              description: provisioningInterface is the name of the network interface
+                on a baremetal server to the provisioning network. It can have values
```

Comment:
This sentence seems awkward to me.  I think I would say "proisioningInterface is the name of the network interface on a baremetal server that is used to provision new machines"

Reply:
Thank you for the comment @imain . I will take care of it in the next MAO patch containing changes to read this CR.